packages/blade/package.json

```diff
@@ -161,7 +161,6 @@
     "tsconfig-paths-webpack-plugin": "3.5.2"
   },
   "peerDependencies": {
-    "@babel/runtime": "^7.16.5",
```

Comment:
this was listed in `dependencies` and `peerDepdencies` both which itself is contradictory 😄. It ideally should be in `dependencies` so consumers dep installer will install it by default